index.src.html

```diff
@@ -91,11 +92,10 @@
       text: creation URL
     urlPrefix: workers.html
       text: running a worker; url: run a worker
-    urlPrefix: dom.html
-      text: document referrer policy; url: concept-document-referrer-policy; for: Document
   type: interface
     urlPrefix: dom.html
       text: Document
+      text: referrer policy; url: concept-document-referrer-policy; for: Document
```

Comment:
This is sounding familiar... https://github.com/tabatkins/bikeshed/issues/786 I guess. Ugh.

Maybe as a workaround

```
    urlPrefix: dom.html
      text: document referrer policy; url: concept-document-referrer-policy; for: Document
```

with `<a for="Document" lt="document referrer policy">referrer policy</a>` will work, although if you do that I'd add a comment about it being a hack.